Python/Product/PythonTools/PythonTools/Project/PythonProjectNode.cs

```diff
@@ -276,11 +277,10 @@ internal class PythonProjectNode :
         }
 
         private void GlobalDefaultInterpreterChanged(object sender, EventArgs e) {
-            // This event is only raised when our active interpreter is the
-            // global default.
-            var evt = ActiveInterpreterChanged;
-            if (evt != null) {
-                evt(this, EventArgs.Empty);
+            if (_active == null) {
```

Comment:
`_active` could use a better name at some point (I realize you didn't introduce it in this PR).